Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -4,7 +4,7 @@
 function BreakBuildOnQualityGateFailure
 {    
     $breakBuild = GetTaskContextVariable "MSBuild.SonarQube.Internal.BreakBuild"
-    $breakBuildEnabled = Convert-String $breakBuild Boolean
+    $breakBuildEnabled = [System.Convert]::ToBoolean($breakBuild)
```

Comment:
Note, Convert-String collides on name with a cmdlet in Microsoft.PowerShell.Utility - so it's good this is gone from our libs now :)

An alternative is to call `Get-VstsTaskVariable -AsBoolean` or `[bool]"$breakBuild"`. What you have here blows up on empty string - just fyi.